src/services/dashboards/dashboard-customize/modules/DashboardCustomizePageName.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <p-page-title child
+                  @goBack="$router.go(-1)"
+    >
+        <template #title>
+            <p-text-input v-if="state.editMode"
+                          v-on-click-outside="handleEscape"
+                          :value.sync="state.nameInput"
+                          @keydown.esc="handleEscape"
+                          @keydown.enter="handleEnter"
+            />
+            <span v-else
+                  class="title-area"
+                  @click="handleClickTitle"
+            >
+                {{ state.name }}
+            </span>
+        </template>
+    </p-page-title>
+</template>
+<script setup lang="ts">
+// Below directive is used. Do not remove!!!
+import { vOnClickOutside } from '@vueuse/components';
+import { computed, reactive, watch } from 'vue';
+
+import { PPageTitle, PTextInput } from '@spaceone/design-system';
+
+const props = defineProps<{ name: string; }>();
+const emit = defineEmits(['update:name']);
```

Comment:
```suggestion
const emit = defineEmits<{(e: string, value: string): void}>();
```
you (maybe) can use `useProxyValue` with this type.

Reply:
Wow it really works!!! GENIUS